homeassistant/components/netatmo/data_handler.py

```diff
@@ -60,6 +62,8 @@
 WEATHER = "weather"
 AIR_CARE = "air_care"
 PUBLIC = NetatmoDeviceType.public
+DOOR_TAG = "opening"
```

Comment:
```suggestion
DOOR_TAG = "door_tag"
```
can this be door_tag instead of opening?

Reply:
No. I tried it. It is related that in pyatmo (netatmo API) door tags are defined as opening entities. As I understood.

Reply:
Then rename it to OPENING?